Update handling-data.tex (#79)
worldbank/d4di: chapters/handling-data.tex

This scrutiny involves two major components: data handling and analytical quality.
Performing at a high standard in both areas means that research participants
are appropriately protected,
and that consumers of research can have confidence in its conclusions.

What we call ethical standards in this chapter is a set of practices for data privacy and research transparency that address these two components.
Their adoption is an objective measure to judge a research product's performance in both.
Without these transparent measures of credibility, reputation becomes the primary signal for the quality of evidence, and hence, two failures may occur:
low-quality studies from reputable sources may be used as evidence when in fact they don't warrant it,
and high-quality studies from sources without an international reputation may be ignored.
Both these outcomes reduce the quality of evidence overall.
Even more importantly, they usually mean that credibility in development research
accumulates at international institutions and top global universities
instead of the people and places directly involved in and affected by it.

> Simple transparency standards mean that it is easier to judge research quality. Additionally, making high-quality research identifiable also increases its impact.

Reviewer: ```suggestion
Simple transparency standards mean that it is easier to judge research quality. Making high-quality research identifiable in turn increases its impact.
```
